policy/slemicro/rke2.fc

```diff
@@ -9,6 +9,7 @@
 /usr/local/lib/systemd/system/rke2.*                                --  gen_context(system_u:object_r:container_unit_file_t,s0)
 /usr/bin/rke2                                                       --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/local/bin/rke2                                                 --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/opt/rke2/bin/rke2                                                  --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
```

Comment:
I think the PR is good to go, but we can't create a context to custom paths including /opt/rke2, can you remove this and I will merge the PR afterwards

Reply:
I think this is still required because of:
```
slemicro53:~ # curl -sfL https://get.rke2.io/ | sh -
[WARN]  /usr/local is read-only or a mount point; installing to /opt/rke2
```

If you install via the rpm method this is bypassed due to transactional-update.

Reply:
```slemicro54:~ # /opt/rke2/bin/rke2
NAME:
   rke2 - Rancher Kubernetes Engine 2

USAGE:
   rke2 [global options] command [command options] [arguments...]

VERSION:
   v1.25.12+rke2r1 (a0aa49e91d86a9a5eec8d94cdab13afabe11bfb6)

COMMANDS:
   server           Run management server
   agent            Run node agent
   etcd-snapshot    Trigger an immediate etcd snapshot
   certificate      Manage RKE2 certificates
   secrets-encrypt  Control secrets encryption and keys rotation
   token            Manage bootstrap tokens
   help, h          Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --help, -h     show help
   --version, -v  print the version```